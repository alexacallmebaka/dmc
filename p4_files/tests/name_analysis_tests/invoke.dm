f : (arg1 : int,arg2 : bool) int {
}

g : () void {
	a : int;
	g();
	f(a, true);
}
