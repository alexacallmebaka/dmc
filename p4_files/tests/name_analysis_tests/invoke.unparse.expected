f{(int,bool)->int} : (arg1{int} : int, arg2{bool}: bool) int {
}

g{()->void} : () void {
    a{int} : int;
    g{()->void}();
    f{(int,bool)->int}(a{int}, true);
}
